### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
As promised, thorough documentation about the usage and development decisions involved.

### CODE REVIEW

**Comment by CodyCBakerPhD** [comment]
(still WIP, will finish this up tomorrow)

**Comment by CodyCBakerPhD** [comment]
OK, ready for review

File: docs/development.rst [hunk 1]
```diff
@@ -1,15 +1,80 @@
...
+
+.. note::
+
+    If this ``results`` folder eventually becomes too large for Git to reasonably handle, we will explore options to share via other data storage services.
+
+
+Network Tracking
+----------------
+
+Please contact Oliver Ruebel for details.
```
Comment (oruebel) [inline docs/development.rst:80]:
```suggestion
Stay tuned https://github.com/NeurodataWithoutBorders/nwb_benchmarks/issues/24
```

**Review by oruebel — APPROVED** [review]
Looks great! Thanks for all the effort writing docs.

**Pull request merged**